app/controllers/api/stories_controller.rb

```diff
@@ -17,6 +17,22 @@ def after_create_resource(res)
     res.create_story_distributions
   end
 
+  def after_update_resource(res)
+    dist_tmpl = res
+      .try(:series)
+      .try(:distributions)
+      .try(:find) { |d| d.kind == "podcast" }
+      .try(:audio_version_template_id)
+
+    version_tmpl = res.try(:audio_versions)
+      .try(:find) { |av| !av.audio_version_template_id.nil? }
```

Comment:
this may be bad feedback (and not a demand for change) as what you've done here is quite readable, but if you did have a list of `dist_tmpl` ids, then you could check here if `dist_tmpl.include?(av.audio_version_template_id)` and save yourself that comparison below.